posthog/batch_exports/models.py

```diff
@@ -263,6 +263,28 @@ def interval_time_delta(self) -> timedelta:
             return timedelta(**kwargs)
         raise ValueError(f"Invalid interval: '{self.interval}'")
 
+    @property
+    def jitter(self) -> timedelta:
+        """Return jitter for this batch export based on interval.
+
+        We always want the start jitter to be less than the frequency, as
+        otherwise a batch export run could start after it's batch period has
+        elapsed. But there is margin to tweak this under that upper limit.
+
+        So, the values set here are quite arbitrary and are adjusted based on
+        how other systems react to batch exports load.
+        """
+        if self.interval == "hour":
+            return timedelta(minutes=15)
+        elif self.interval == "day":
+            return timedelta(hours=1)
+        elif self.interval == "week":
+            return timedelta(days=1)
```

Comment:
this one might be a bit large but I don't think we actually have any weekly batch exports (?) so is fine

Reply:
Yeah it's mostly there as it is one of the supported options in our model, so just covering all potential cases.

Reply:
But, like you said, we don't currently expose this as a frequency, so none exist.

Reply:
Keep in mind 1 day should be smaller than the current jitter, as 1 week / 6 > 1 day!